### ISSUE TITLE
Open Crop Icons

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/icons#5]
We're starting to do some work on open (public domain) icons for some popular crops. 

If you want to design some neat icons for plants, have a look here:

https://github.com/openfarmcc/open-crop-icons/tree/master

original conversation here:

https://github.com/openfarmcc/OpenFarm/issues/685

original OSD job listing here: http://opensourcedesign.net/jobs/jobs/2015-09-SVG-plant-icons.html

**Comment by jancborchardt** [comment]
I’d say it’s probably best if we submit any of our icons to https://thenounproject.com instead of replicating stuff here. Since we now have these icon pages linked on our website too, so we can close this repo?

**Comment by ei8fdb** [comment]
Agreed, but maybe in the README point towards an "opensourcedesign" account covering all the icons on the thenounproject?

Gimme some thumbs up and I'll create the account, and then share the user/pass?

**Comment by jancborchardt** [comment]
I don’t see the need for a specific opensourcedesign account. All of The Noun Project is open source design, and I suppose everyone of us has individual accounts. Open Source Design is more than just the icons of specific people. ;)

**Comment by ei8fdb** [comment]
I have no strong feelings for or against the creation. :) Just a thought.

**Comment by jancborchardt** [comment]
@simonv3 @bnvk @johnhbenetech @jhermann as said above:

> I’d say it’s probably best if we submit any of our icons to https://thenounproject.com instead of replicating stuff here. Since we now have these icon pages linked on our website too, so we can close this repo?

Cause basically at the moment we’re just doing yet another icon platform (as a repo only that is), while linking to The Noun Project and other platforms for open source icons is much more impactful.

**Comment by bnvk** [comment]
I *think* the contents of the repo + what I wrote in the README is slightly different than *"just doing yet another icon platform"*, please [read it](https://github.com/opensourcedesign/icons/blob/master/README.md):

> The loose idea of this repo is to apply aspects of open source software
> development (sharing, iteration, forking, extending, and modifying) to the
> icon design process.

I also outlined "Getting Started" steps... I don't think building "yet another icon platform" is anything I wrote or am advocating for- see the bottom of the README:

> However, must be careful not to replicate The Noun Project

To date, we have had 3 pull-requests of icon submissions. Sadly, 2 of them have sat waiting to be approved for 2+ yrs. Maybe no one else understood, see's value in what I was thinking. Clearly, no one was paying attention to the pull-requests... even now.

I've been meaning to upload all of the icons ever designed, as I kind of despise that one has to login with an NounProject account in order to download some open source icons/code and I'd like FOSS designers to be able to just get my icons free of hassle.

As far as having an OSD account on NP... well... the benefit would be when people working at ad agencies and corporations D/L your icon- you get paid :money_mouth_face: feel free to consider and discuss what this could/would mean for OSD / contributors...

**Comment by raucao** [comment]
> I've been meaning to upload all of the icons ever designed, as I kind of despise that one has to login with an NounProject account in order to download some open source icons/code and I'd like FOSS designers to be able to just get my icons free of hassle.

👍 👏

**Comment by jancborchardt** [comment]
> As far as having an OSD account on NP... well... the benefit would be when people working at ad agencies and corporations D/L your icon- you get paid :money_mouth_face: feel free to consider and discuss what this could/would mean for OSD / contributors...

Well, that’s why I’d say everyone should have an individual account – that’s what I do. If we have a grouped account, it raises questions how to split it etc. (sure we could put it into OpenCollective. If anyone wants to put work in that sure, however I don’t see the impact there.)
